cylc/flow/task_events_mgr.py

```diff
@@ -965,6 +976,44 @@ def _process_message_check(
         LOG.log(severity_lvl, f"[{itask}] {flag}{message}{timestamp}")
         return True
 
+    def process_job_message(
+        self,
+        job_tokens: 'Tokens',
+        tdef: 'TaskDef',
+        message: str,
+        event_time: str,
+    ) -> bool:
+        """Process a job message only.
```

Comment:
I'm not clear about the "only"?
Can you say what it's not doing?

```suggestion
        """Process a job message without altering job state.
```